### DISCUSSION TITLE
Why using 'commonName' is disabled?

### DISCUSSION THREAD

**Discussion by DiskoRico** [encode/httpx#3125] [Q&A] [answered]
I am using "old" self signed certificates that does not include `subjectAltName`.
I don't want to generate new ones.

1. Why this behavior is disabled in first place?
2. Is there a possibility to add an option to override `context.hostname_checks_common_name`?
https://github.com/encode/httpx/blob/7e10342c2ad44955582e4fef96470f3e653347d4/httpx/_config.py#L139-L144

Thanks!

**Comment by DiskoRico** [comment]
I eventually used `create_ssl_context` from _config to create ssl context and change `hostname_checks_common_name` to True.
After that I have passed the context to my call and it worked fine 😄

**Reply by lovelydinosaur** [reply, marked as answer]
This'll be more obvious once we get this in... https://github.com/encode/httpx/pull/3022

**Reply by DiskoRico** [reply]
Thanks for the fast response 😄
